extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,17 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+.amp-story-shopping-product-tag-text {
+  font-family: 'Poppins','Arial', sans-serif !important;
+  font-weight: 600 !important;
+  font-size: 11px !important;
+  line-height: 15.4px !important;
+  font-style: normal;  
```

Comment:
There's a trailing whitespace at the end of this line that's causing the invalid whitespace check to fail

Reply:
`normal` is the default font-style value, so I think this line may be unnecessary.